tests/common/plugins/pdu_controller/controller_base.py

```diff
@@ -1,56 +1,56 @@
 """
-Base class for controlling PSUs of DUT
+Base class for controlling PDUs conecting to DUT
 
-This file defines the base class for controlling PSUs of DUT. The base class defined the basic interface of
-PSU controllers.
+This file defines the base class for controlling PDUs outlets. The base class defined the basic interface of
+PDU controllers.
 
-The PSU controller for actually controlling PSUs must be a subclass of the PsuControllerBase class and must
+The PDU controller for actually controlling PDUs must be a subclass of the PduControllerBase class and must
```

Comment:
Suggest removing `for actually controlling PDUs`

Reply:
Thanks Joe! Fixed.